empathylink.lic

```diff
@@ -0,0 +1,130 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#empathylink
+=end
+
+custom_require.call(%w[common drinfomon equipmanager])
+
+class Empathylink
+  def initialize
+    if UserVars.empathylink.nil?
+      UserVars.empathylink=Hash.new
+    end
+    populate_noheal_list
+    settings = get_settings
+    @buddies = settings.hunting_buddies
+    @use_hodierna = settings.has_hodierna
+    in_room = DRRoom.pcs
+    while true
```

Comment:
This is going to loop forever as far as i can tell. Do you want this? **documentation nudge**

Reply:
I like to use the pull request description as a first draft of documentation - you can write it there if inclined. Highly recommended if you respect reviewers time.

Reply:
Yes, we want it to loop because you start it and let it run. Then, when hunting, it will check if buddies are in the room, make a link, and start the healing.

Reply:
Do you want it to check if leaves the room or anything? seems risky if any random script triggers that causes you to move to the bank or somesuch.

Reply:
Although, I am going to change the no heal list from populating here to inside your yaml.